src/SprykerEco/Shared/Unzer/UnzerConfig.php

```diff
@@ -14,20 +14,235 @@ class UnzerConfig extends AbstractBundleConfig
     /**
      * @var string
      */
-    public const PROVIDER_NAME = 'unzer';
```

Comment:
Shared config is a part of public API, so you can't just delete an old const, instead just deprecate it (see https://docs.spryker.com/docs/scos/dev/architecture/module-api/definition-of-module-api.html)
Of course, it's not the case if you are planning to have a major release.

Reply:
It's major release ;)